tests/func-tests/monitoring_test.go

```diff
@@ -109,15 +104,49 @@ var _ = Describe("[crit:high][vendor:cnv-qe@redhat.com][level:system]Monitoring"
 		}
 		updateHCO(virtCli, hco)
 
-		Eventually(func() *promApiv1.Alert {
-			alerts, err := promClient.Alerts(context.TODO())
+		verifyAlertExists(promClient, "KubevirtHyperconvergedClusterOperatorUSModification")
+	})
+
+	It("KubeVirtHyperconvergedClusterOperatorPodSchedulingMightBeImbalanced alert should fired when node max images is reached", func() {
+		nodes, err := virtCli.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+		Expect(err).ShouldNot(HaveOccurred())
+		Expect(nodes.Items).ShouldNot(BeEmpty())
+
+		node := &nodes.Items[0]
+		maxImages, err := getNodeMaxImages(virtCli, node.Name)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		if maxImages == -1 {
+			Skip("maxImages is -1, skip test")
+		}
+
+		dhil := initDockerHubImageList("busybox", "1.")
+
+		pod := createPod(virtCli, node, dhil.getNextImage())
```

Comment:
in this test, we need to make sure the node has as many images as the limit, usually 50. My idea was to iterate the busybox images in DockerHub and have a pod going through all those images to download them to the node. Couldn't think of any easier way, any suggestions here also welcomed